### ISSUE TITLE
allow_alias option ignored

### ISSUE THREAD

**Issue by jonyscathe** [yoheimuta/protolint#495]
As of 0.56.x the option allow_alias = true is being ignored.

For legacy API reasons I require some aliases in a an enum.

Example:

```
enum MyEnumType {
  option allow_alias = true;
  MY_ENUM_TYPE_UNSPECIFIED = 0;
  MY_ENUM_OPTION_1 = 1;
  MY_ENUM_OPTION_1_ALT = 1;
  MY_ENUM_OPTION_2 = 2;
  MY_ENUM_OPTION_3 = 3;
}
```

In 0.55.6 and earlier, this was always fine.
In 0.56.1 this reported:
```
fields  and MY_ENUM_TYPE_UNSPECIFIED have the same number 0
fields MY_ENUM_OPTION_1 and MY_ENUM_OPTION_1_ALT have the same number 
```
In 0.56.2 and 0.56.3 this reported:
```
fields MY_ENUM_OPTION_1 and MY_ENUM_OPTION_1_ALT have the same number 
```

I would expect the allow_alias option to be recognised and have no error thrown here.
Is this what should be happening or do I need to add a protolint:disable comment here?
